Documentation/devel/stage1-implementors-guide.md

```diff
@@ -28,31 +28,31 @@ This allows users to employ different containment strategies on the same host ru
+0. rkt prepares the pod's stage 1 and stage 2 images and Pod Manifest under `/var/lib/rkt/pods/$uuid`, acquiring an exclusive advisory lock on the directory
+1. chdirs to `/var/lib/rkt/pods/$uuid`
+2. resolves the `coreos.com/rkt/stage1/run` entrypoint via Annotations found within `/var/lib/rkt/pods/$uuid/stage1/manifest`
+3. executes the resolved entrypoint relative to `/var/lib/rkt/pods/$uuid/stage1/rootfs`
 
 It is the responsibility of this entrypoint to consume the Pod Manifest and execute the constituent apps in the appropriate environments as specified by the Pod Manifest.
 
-The environment variable "RKT_LOCK_FD" contains the file descriptor number of the open directory handle for "/var/lib/rkt/pods/$uuid".
+The environment variable `RKT_LOCK_FD` contains the file descriptor number of the open directory handle for `/var/lib/rkt/pods/$uuid`.
 It is necessary that stage 1 leave this file descriptor open and in its locked state for the duration of the `rkt run`.
 
-In the bundled rkt stage 1 which includes systemd-nspawn and systemd, the entrypoint is a static Go program found at "/init" within the stage 1 ACI rootfs.
+In the bundled rkt stage 1 which includes systemd-nspawn and systemd, the entrypoint is a static Go program found at `/init` within the stage 1 ACI rootfs.
 The majority of its execution entails generating a systemd-nspawn argument list and writing systemd unit files for the constituent apps before executing systemd-nspawn.
 Systemd-nspawn then boots the stage 1 systemd with the just-written unit files for launching the contained apps.
-The "/init" program is essentially a Pod Manifest to systemd-nspawn + systemd.service translator.
+The `/init` program is essentially a Pod Manifest to systemd-nspawn + systemd.service translator.
 
 An alternative stage 1 could forego systemd-nspawn and systemd altogether, or retain these and introduce something like novm or qemu-kvm for greater isolation by first starting a VM.
-All that is required is an executable at the place indicated by the "coreos.com/rkt/stage1/run" entrypoint which knows how to apply the Pod Manifest and prepared ACI file-systems to good effect.
+All that is required is an executable at the place indicated by the `coreos.com/rkt/stage1/run` entrypoint which knows how to apply the Pod Manifest and prepared ACI file-systems to good effect.
 
 The resolved entrypoint must inform rkt of its PID for the benefit of `rkt enter`.
 rkt supports two ways of doing that, stage1 implementors must pick one:
 
-1. Writing to "/var/lib/rkt/pods/$uuid/pid" the PID of the process that is PID 1 in the container.
-2. Writing to "/var/lib/rkt/pods/$uuid/ppid" the PID of the parent of the process that is PID 1 in the container.
+1. Writing to `/var/lib/rkt/pods/$uuid/pid` the PID of the process that is PID 1 in the container.
+2. Writing to `/var/lib/rkt/pods/$uuid/ppid` the PID of the parent of the process that is PID 1 in the container.
```

Comment:
* `/var/lib/rkt/pods/$uuid/ppid`: the PID of the parent of the process that is PID 1 in the container